src/test/java/com/training/web/tests/flipkart/SocialMediaTest.java

```diff
@@ -0,0 +1,40 @@
+package com.training.web.tests.flipkart;
+
+import com.training.basetest.WebBaseTest;
+import com.training.web.pages.flipkart.FlipkartHomePage;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvFileSource;
+import static com.codeborne.selenide.Selenide.open;
+
+public class SocialMediaTest extends WebBaseTest {
+    FlipkartHomePage flipkartHomePage;
+
+    @BeforeEach
+    public void setup() {
+        open("https://www.flipkart.com/");
+        flipkartHomePage = new FlipkartHomePage().closePopup();
+    }
+
+    @ParameterizedTest
+    @CsvFileSource(resources = "/testSocialMediaPage.csv")
+    public void testSocialMediaPage(String media) throws InterruptedException {
```

Comment:
Lets add one more parameter - expectedMediaUrl -> and pass this from csv file, instead of hardcoding in the switch loop.